config.rb

```diff
@@ -165,13 +165,24 @@
   redirect "v#{version}/guides/bundler_2_upgrade.html", to: "guides/bundler_2_upgrade.html"
 end
 
+# Redirect to the latest man page when version isn't specified
+man_files = Dir.glob("./source/#{config[:current_version]}/man/*.html.erb")
+generic_man_urls = man_files.map do |file|
+  file.delete_prefix!("./source/#{config[:current_version]}/")
+  file.delete_suffix!(".erb")
+end
+generic_man_urls.each do |url|
+  latest_man = "#{config[:current_version]}/#{url}"
```

Comment:
Does this simplification make sense?

```suggestion
  url = file.delete_prefix("./source/#{config[:current_version]}/").delete_suffix(".erb")
  latest_man = "#{config[:current_version]}/#{url}"
```

Reply:
Yeah, a lot better.